### PR TITLE
[OCDM] Apply changes in decrypt()'s iface

### PR DESCRIPTION
**Pull request by pawel-stanek-ml**

### CODE REVIEW

File: Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp
Comment (calvaris) [inline Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp:206]: What's the trailing 0?

**Comment by pawel-stanek-ml** [comment]
An additional param added by netflix (uint32_t initwithLast15). Irrelevant for webkit and 0 shouldbe used in that case.

File: Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp
Comment (pawel-stanek-ml) [reply Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp:206]: An additional param added by netflix (uint32_t initwithLast15). Irrelevant for webkit and 0 shouldbe used in that case.

**Review by calvaris — APPROVED** [review]

File: Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp
Comment (charlie-ht) [reply Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp:206]: I recommend refactoring this interface to take a struct parameter, so that random bits of information like that can be properly named and more easily read by maintainers.

Comment (pawel-stanek-ml) [reply Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp:206]: It's now slightly better then before (due to reduction of number of params, and dropping cast). Unfortunately this time it was a coordinated with netflix guys (they had to update too) and I didn't want to force them to do another round.

Comment (charlie-ht) [reply Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp:206]: I'd say this is a case of "Short-term pain, long-term gain.", but I won't flog the horse any further, I leave it to your discretion.

**Pull request merged**
